Packages/ChatSecure/ChatSecure/Classes/WebRTCJanus/WebRTC+Custom/RTCSimulatorVideoDecoderFactory.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  RTCSimulatorVideoDecoderFactory.swift
+//  GoogleWebRTCJanus
+//
+//  Created by VietAnh on 12/23/20.
+//  Copyright © 2020 Vmodev. All rights reserved.
+//
+
+import Foundation
+import WebRTC
+
+class RTCSimulatorVideoDecoderFactory: RTCDefaultVideoDecoderFactory {
+	override init() {
+		super.init()
+	}
+	
+	override func supportedCodecs() -> [RTCVideoCodecInfo] {
+		var codecs = super.supportedCodecs()
+		codecs = codecs.filter { $0.name != "H264" }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/1ab8931f0f28255ebeb64e047f8b06b326be6a2d/Packages/ChatSecure/ChatSecure/Classes/WebRTCJanus/WebRTC+Custom/RTCSimulatorVideoDecoderFactory.swift#L19"></span>Override methods which only call super can be removed</td>
    </tr>
  </tbody>
</table>